perception_dataset/rosbag2/converter_params.py

```diff
@@ -97,6 +97,9 @@ def __init__(self, **args):
                 "The config of `camera_sensors` field is empty, so disable to load camera data."
             )
             self.with_camera = False
+        for sensor in self.camera_sensors:
+            if not hasattr(sensor, "delay_msec"):
+                sensor["delay_msec"] = 0.0
```

Comment:
[must] 
Because of this, every delay for the camera sensor in every conversion is set to 0, which is not the expected behavior.
Please fix the logic.
![image](https://github.com/user-attachments/assets/45c2cf8b-6c74-46f2-93a8-6870aa50a9cd)

Reply:
I fixed in https://github.com/tier4/tier4_perception_dataset/pull/140/commits/fe1b03c10623c657f46d3759201047ce9c68af7c
 and I checked output data is correct in local DLR.